docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3"
+networks:
+  intergration-test:
+    driver: bridge
+services:
+  postgres:
+    image: postgis/postgis
+    container_name: postgres
+    restart: on-failure
+    healthcheck:
+      test: /usr/bin/pg_isready -U postgres
+      interval: 5s
+      timeout: 10s
+      retries: 120
+    ports:
+      - 5432
+    env_file:
+      - ./postgres/postgres.env
+    volumes:
+      - pg_data:${PGDATA:-/var/lib/postgresql/data}:rw
+      - ./postgres/01-init-user.sh:/docker-entrypoint-initdb.d/01-init-user.sh
+    networks:
+      - intergration-test
+
+  geoserver:
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+      args:
+        GEOSERVER_WEBAPP_SRC: "https://build.geoserver.org/geoserver/master/geoserver-master-latest-war.zip"
```

Comment:
master branch has been renamed to "main"

Reply:
solved here https://github.com/geosolutions-it/docker-geoserver/pull/43/commits/329db3edae5f726910e2dc637206841b6e51af9a